### PR TITLE
Update js.tpl

### PR DESCRIPTION
**Pull request by wdammak**
PHP 7.2 & 7.3

### CODE REVIEW

**Review by shreesh-webkul — COMMENTED** [review]
Should use empty, is_array it will allow empty array to pass-through

**Comment by wdammak** [comment]
> 
> 
> Should use empty, is_array it will allow empty array to pass-through

PHP >=7.2 warning - count(): Parameter must be an array or an object that implements Countable

File: admin/themes/default/template/controllers/modules/js.tpl [hunk 1]
```diff
@@ -34,7 +34,7 @@
 	var currentIndexWithToken = '{$currentIndex|escape:'html':'UTF-8'}&token={$token|escape:'html':'UTF-8'}';
 	var dirNameCurrentIndex = '{$dirNameCurrentIndex}';
 	var ajaxCurrentIndex = '{$ajaxCurrentIndex}';
-	var installed_modules = {if isset($installed_modules) && count($installed_modules)}{$installed_modules}{else}false{/if};
+	var installed_modules = {if isset($installed_modules) && is_array($installed_modules) && count($installed_modules) > 0}{$installed_modules}{else}false{/if};
```
Comment (shreesh-webkul) [inline admin/themes/default/template/controllers/modules/js.tpl]: No need to compare `count($installed_modules) > 0`.

Comment (shreesh-webkul) [reply admin/themes/default/template/controllers/modules/js.tpl]: @wdammak 
waiting for an update.

Comment (wdammak) [reply admin/themes/default/template/controllers/modules/js.tpl]: @shreesh-webkul done.

Comment (shreesh-webkul) [reply admin/themes/default/template/controllers/modules/js.tpl]: @wdammak 
There has been a misunderstanding. 
Actually, we need to check `count($installed_modules)`, the changes required were only to remove its comparison with 0.
It should be
`{if isset($installed_modules) && is_array($installed_modules) && count($installed_modules)}`

Comment (wdammak) [reply admin/themes/default/template/controllers/modules/js.tpl]: @shreesh-webkul done.

Comment (shreesh-webkul) [reply admin/themes/default/template/controllers/modules/js.tpl]: @wdammak prefect!.

**Review by shreesh-webkul — APPROVED** [review]
Changes accepted.

**Pull request merged**